### DISCUSSION TITLE
last call for drinks

### DISCUSSION THREAD

**Discussion by priv-kweihmann** [priv-kweihmann/meta-sca#8723] [Announcements]
if you want this layer to stay up to date and given for free with the current scope and support for upstream layers, you could do one of the following

- provide some computing power to keep the maintenance up in terms of quality and speed
- provide feedback on your use cases
- provide financial support for the vast of time I'm (as a single maintainer) putting into all that

Lately working on this layer was making less and less fun - tons of upstream changes to be integrated, changed, tested, a.s.o., support for tickets has to be done - please keep in mind that all this is done in my spare time on my own equipment without any company support, nor financial gain of whatsoever - it's more of the case that I actually spend money to keep this here up and running.

If there's no motivation to get from working on it, I have to consider changes to the way this layer is maintained.
In the end that would mean

- less actively supported branches (basically only the latest release branch - at the time of writing honister - or whatever branch I need for my work or personal projects)
- no more tag releases
- higher reaction time on tickets (if there will be any at all)
- less documentation
- way less updates
- drastically reduced amount of features (basically cut down to the amount I personally really use)
- axed CI support for daily integration

After some thought I see no other option but to make it a personal fun time project again or to keep on working with a reasonable amount of help of (according to the stats) the plenty of users out there.

In the end it's your choice as an user :smile: 

I will leave this here open to be commented on - and if there isn't any response the outlined option will be become effective without further notice.

I don't want to become rich or famous with this work here, but at least it should be a bit more of fun - and I honestly think that's easy to achieve

Feel free to reach out

**Comment by SJLC** [comment]
I'm guessing most people don't ever look at discussions under github unless they have a burning question to ask. Maybe you should give a brief mention of these points near the top of the README in master branch, which a lot more users are likely to at least look at? Then point here, for those who actually want to contribute in some way.

**Reply by SJLC** [reply]
Also another hint for making it more obvious where people can help out, could also add a link in the README to https://github.com/priv-kweihmann/meta-sca/issues?q=is%3Aissue+is%3Aopen+-label%3A%22Package+Update%22
which filters out the auto-generated issues, to highlight things like your announcements and user questions that otherwise get buried

**Reply by priv-kweihmann** [reply]
I get your point about issues being lost due to package update tickets, but the issue itself is pinned (as any other ticket that would need help from the community) - so it should already have the needed visibility.
Regarding the README I have to admit that I've never seen that people read them once this layer is integrated, so I don't really see the point - also because the README is already quite big at the moment.

What I will do is to add documentation how people can participate in maintaining one of the branches - BTW I started discussions, I pinned tickets, I posted on LinkedIn and still I think most of the users just don't see this upcoming change or are just not able to participate in this discussion here.

The fact that you're the first user to post here after ~1.5 months is saddening for me.

**Comment by SJLC** [comment]
I'm basically extrapolating from my own experience in above recommendations, in that a team I work on uses meta-sca so I knew it was here, but I didn't notice the pinned issues at all. Only found this after doing above search to see if there was any issues about recent master breakage, so I could pitch in while in master-fallout-fixing mode.

What can I say, guess I'm not super observant when I'm focused on a task at hand, maybe I would not have noticed a big NOTE sy the top of the front page README either ;)

**Reply by priv-kweihmann** [reply]
Same here :grin: - and thanks for helping out

**Reply by SJLC** [reply]
Feel free to @-mention me when (not 'if' `;)` ) there is more breakage on master, I'll help if I can
